### PR TITLE
face-services UI

### PR DESCRIPTION
**Pull request by ferrouswheel**
- [x] face-detect
- [x] face-landmarks
- [x] face-align
- [x] face-identity

### CODE REVIEW

**Comment by ferrouswheel** [comment]
image

**Comment by ferrouswheel** [comment]
image

**Comment by ferrouswheel** [comment]
image

**Comment by ferrouswheel** [comment]
Face Identity

image

**Comment by ferrouswheel** [comment]
Ready for review/merge.

File: src/components/service/FaceDetectService.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ export default class FaceDetectService extends React.Component {
             setTimeout ( () => this.renderBoundingBox(result), 200 );
             return;
         }
-        let desiredWidth = 300.0; // TODO: find appropriate reference width from components
+        let desiredWidth = 500.0; // TODO: find appropriate reference width from components
```
Comment (tesYolan) [inline src/components/service/FaceDetectService.js]: There is this.props.sliderWidth property you can use to see the size modal component. See https://github.com/tesYolan/snet-dapp/blob/upstream/image-services/src/components/service/emotion-recognition-service/EmotionVisualizer.js#L64

File: src/components/service/FaceAlignService.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            }
+            this.setState({inputValid: this.checkValid()});
+        }
+        if (this.state.methodName !== prevState.methodName)
+            this.setState({inputValid: this.checkValid()});
+        if (this.state.imageData !== prevState.imageData)
+            this.setState({inputValid: this.checkValid()});
+      }
+
+    componentWillReceiveProps(nextProps) {
```
Comment (raamb) [inline src/components/service/FaceAlignService.js]: WIth pr #170 merged in these components can be cleaned up. Did you plan on doing that in a subsequent iteration? (which is fine too)

**Comment by ferrouswheel** [comment]
Uses sliderWidth and removes all the parser stuff. Should be okay to merge now.

**Comment by ferrouswheel** [comment]
Should be good to go now.

**Pull request merged**